pkg/common/resources/resources_test.go

```diff
@@ -588,6 +588,46 @@ func TestComponentWiseMinOnlyExisting(t *testing.T) {
 	}
 }
 
+func TestComponentWiseMinPermissiveWithPrecedence(t *testing.T) {
```

Comment:
This test name makes no sense. You're not calling that method from the test at all.